homework-custom-lodash/includes.js

```diff
@@ -0,0 +1,42 @@
+const includes = function(collection, value, fromIndex) {
+  let n = fromIndex || 0;
+  function sameValueZero(x, y) {
```

Comment:
please move this function in the closure, because now every time when u will execute this func u will create a new `sameValueZero` func